multispaeti/_multispati_pca.py

```diff
@@ -46,27 +52,24 @@ class MultispatiPCA:
         A distance matrix should be transformed to connectivities by e.g.
         calculating :math:`1-d/d_{max}` beforehand.
 
-    Raises
-    ------
-    ValueError
-        If connectivity is not a square matrix.
-    ZeroDivisionError
-        If one of the observations has no neighbors.
-
     Attributes
     ----------
     components_ : numpy.ndarray
         The estimated components: Array of shape `(n_components, n_features)`.
 
+    eigenvalues_ : numpy.ndarray
+        The eigenvalues corresponding to each of the selected components. Array of shape
+        `(n_components,)`.
+
     variance_ : numpy.ndarray
         The estimated variance part of the eigenvalues. Array of shape `(n_components,)`.
 
     moransI_ : numpy.ndarray
         The estimated Moran's I part of the eigenvalues. Array of shape `(n_components,)`.
 
-    eigenvalues_ : numpy.ndarray
-        The eigenvalues corresponding to each of the selected components. Array of shape
-        `(n_components,)`.
+    mean_ : numpy.ndarray or None
+        Per-feature empirical mean, estimated from the training set if `X` is not sparse.
```

Comment:
is it igored in case of non sparse data??

Reply:
then maybe you should update it to write that

Reply:
Good point. But this probably should be documented in `fit`/`transform`?

Reply:
i think it should be documented where it is declared